### PR TITLE
Datasets lazy load

### PR DESCRIPTION
**Pull request by kinegratii**
This PR makes coordinate data be lazy load from external store.

Note that `pyecharts.utils` module now is a mixed collections of some tool functions & classes,it should be refactored as a package with different modules.

### CODE REVIEW

File: pyecharts/utils.py [hunk 1]
```diff
@@ -75,3 +79,45 @@ def _add(_items):
...
+    def inner(self, *args):
+        if self._wrapped is None:
+            self._setup()
+        return func(self._wrapped, *args)
+
+    return inner
+
+
+class LazyObject(object):
+    _wrapped = None
```
Comment (chfw) [inline pyecharts/utils.py]: maybe place these two functions in lazy_object.py ?

Comment (chenjiandongx) [reply pyecharts/utils.py]: 感觉没这个必要，这实际上就是 "工具函数"，来自 Django 应该是，小伟以前发过一次。

File: pyecharts/utils.py [hunk 2]
```diff
@@ -75,3 +79,45 @@ def _add(_items):
...
     return front_must_items + fol + dependencies
+
+
+# ------- Lazy Module ------#
+
+def proxy_method(func):
+    def inner(self, *args):
+        if self._wrapped is None:
+            self._setup()
+        return func(self._wrapped, *args)
```
Comment (chenjiandongx) [inline pyecharts/utils.py]: @kinegratii 这行中 把 self._wrapped 加入到 func 的意思是？

Comment (chenjiandongx) [reply pyecharts/utils.py]: 我好像明白了，self._wrapped 其实就是封装了坐标的一个字典，实际上的 get() 方法就是从 wrapped 中获取的。

File: pyecharts/utils/lazy.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# coding=utf8
```
Comment (chfw) [inline pyecharts/utils/lazy.py]: :+1:

**Comment by kinegratii** [comment]
@chfw @chenjiandongx 

This PR can be merged now if no any problem  found.

**Pull request merged**